cmd/litestream/main.go

```diff
@@ -349,6 +354,22 @@ func NewReplicaFromConfig(c *ReplicaConfig, db *litestream.DB) (_ *litestream.Re
 	if v := c.ValidationInterval; v != nil {
 		r.ValidationInterval = *v
 	}
+	for _, str := range c.AgeIdentities {
+		identities, err := age.ParseIdentities(strings.NewReader(str))
+		if err != nil {
+			return nil, err
+		}
+
+		r.AgeIdentities = append(r.AgeIdentities, identities...)
```

Comment:
Why the `append()` instead of just assigning the `identities`? Same goes for the `recipients` below.

Reply:
`age.ParseIdentities` and `age.ParseRecipients` work with NL terminated lists and in the config format I chose to use real YAML lists instead of a multi line string to store them, hence need to loop and append them when parsed one by one.

If preferred I can change both to `string` from `[]string`, then it's just an assignment.

Reply:
Ah, sorry, I misread it. Yeah, that's fine. 👍